web/flat-web/src/utils/i18n.ts

```diff
@@ -19,6 +19,10 @@ void i18next
         interpolation: {
             escapeValue: false, // react already safes from xss
         },
+        detection: {
+            // don't remember user language
+            caches: [],
+        },
```

Comment:
split into another PR

```suggestion
        detection: {
            // no recording of user-selected languages
            caches: [],
        },
```

Reply:
#1269